scripts/fires/wfigs_data.py

```diff
@@ -93,6 +116,11 @@ def get_datetime(x):
         else:
             return None
 
+    def get_date_str(x):
+        if not pd.isna(x):
+            return x.strftime("%Y-%m-%d")
+        return None
+
     # Get location associated with each incident.
     def get_place(x):
```

Comment:
please don't use `x` -- what is a more self-documenting name you could use here? (same for the other helpers here).

Reply:
done